src/main/java/nl/dtls/fairdatapoint/api/controller/exception/ExceptionHandlerAdvice.java

```diff
@@ -52,8 +52,8 @@
 @ControllerAdvice
 public class ExceptionHandlerAdvice {
     
-    private final static Logger LOGGER
-            = LogManager.getLogger(ExceptionHandlerAdvice.class.getName());
+    private final static Logger LOGGER = LoggerFactory.getLogger(
+            ExceptionHandlerAdvice.class.getName());
```

Comment:
We can use the class itself instead of resolving it per name. (also for consistency)